storybookjs/storybook
# Stop inserting HTML tags from error messages

Issue: https://github.com/storybookjs/storybook/issues/8362

## What I did

I have added `escapeXML: true` option to AnsiToHtml constructor call.

### Before
![storybook-html-1](https://user-images.githubusercontent.com/15332326/66521024-a0a62280-eaea-11e9-92df-9b60e7958a9a.png)

### After
![storybook-html-2](https://user-images.githubusercontent.com/15332326/66521046-aa2f8a80-eaea-11e9-9f7b-571c4eca39d3.png)

## How to test

- Is this testable with Jest or Chromatic screenshots?

It's testable with Jest and Chromatic. I didn't add any test yet, because tests on next fail on my machine (Windows 😢) and to run them at all I had to modify https://github.com/storybookjs/storybook/blob/746e98f412ba59e6136a6de808c74a0598c17c27/scripts/test.js#L170 to `const jest = path.join(__dirname, '..', 'node_modules', 'jest', 'bin', 'jest.js');`
~I'm gonna clone the repo on other OS tomorrow and add one.~
Edit: ☝ this is harder than I expected. I'm _trying_ to add a test, I'm gonna commit once I get it done I suppose
